lib/vrl/stdlib/src/parse_duration.rs

```diff
@@ -164,9 +164,16 @@ mod tests {
             tdef: TypeDef::new().fallible().float(),
         }
 
-        error_us {
+        us_s {
+            args: func_args![value: "1 µs",
+                             unit: "ms"],
+            want: Ok(0.001),
+            tdef: TypeDef::new().fallible().float(),
+        }
+
+        error_invalid {
             args: func_args![value: "foo",
-                             unit: "µs"],
+                             unit: "ms"],
```

Comment:
I can't figure out why this change was required 🤔

Reply:
Aha, yeah, it isn't required. I just did it before making the other change to ensure this one was failing because `foo` is not a duration rather than `µs` was not a valid unit.